### PR TITLE
Adding Websocket blog post

### PR DESCRIPTION
**Pull request by matzew**
As per title. Following up on https://github.com/knative/docs/pull/6109

### CODE REVIEW

**Review by Cali0707 — COMMENTED** [review]
One small nit otherwise LGTM!

**Comment by matzew** [comment]
/hold 
for @skonto

File: blog/docs/articles/Knative-Serving-WebSockets.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+websocket-server-00001-deployment-f785cbd65-prjmj   2/2     Running   0          20s
+websocket-server-00001-deployment-f785cbd65-v696v   2/2     Running   0          18s
+websocket-server-00001-deployment-f785cbd65-ws5k9   2/2     Running   0          20s
+websocket-server-00001-deployment-f785cbd65-xmszx   2/2     Running   0          18s
+websocket-server-00001-deployment-f785cbd65-znhrr   2/2     Running   0          20s
+```
+
+!!! note
+
+    Depending on the target annotation you have ([`autoscaling.knative.dev/target`](https://knative.dev/docs/serving/autoscaling/autoscaling-targets/)) you can scale based on num of connections.
```
[the remark this answers is not in the document]
Comment (matzew) [reply blog/docs/articles/Knative-Serving-WebSockets.md:196]: Let's just keep it as is.

**Pull request merged**